plugins/CoreAdminHome/templates/optOut.twig

```diff
@@ -69,7 +69,7 @@
 
     {% if not showConfirmOnly %}
     {% set loadInNewWindow = isSafari and trackVisits %}
-    <form method="post" action="?module=CoreAdminHome&amp;action=optOut{% if language %}&amp;language={{ language }}{% endif %}{% if loadInNewWindow %}&amp;setCookieInNewWindow=1{% endif %}" {% if loadInNewWindow %}target="_blank"{% endif %}>
+    <form method="post" action="?{{ queryParameters }}" {% if loadInNewWindow %}target="_blank"{% endif %}>
```

Comment:
You're right. :+1: 
See: http://twig.sensiolabs.org/doc/filters/url_encode.html